modules/client-configuration/nav-client-configuration-guide.adoc

```diff
@@ -20,39 +20,64 @@ endif::[]
 ifeval::[{uyuni-content} == true]
 *** xref:supported-features-debian.adoc[Debian Supported Features]
 endif::[]
-// Manual Registration
-** xref:registration-overview.adoc[SLE Client Registration]
+// Client concepts
+** xref:channels.adoc[Channels]
+*** xref:repositories.adoc[Repositories]
+*** xref:products.adoc[Products]
+** xref:gpg-keys.adoc[GPG Keys]
+** xref:activation-keys.adoc[Activation Keys]
+** xref:bootstrap-repository.adoc[Bootstrap Repository]
+endif::[]
```

Comment:
do we have a lonely endif here?  Later i see more of them.

Reply:
i think you might be right.